Services/Calendars/CalendarService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Calendars;
+
+namespace Andculture.Sitefinity.Testing.Services.Calendars
+{
+    public class CalendarService : ODataService<Calendar>
+    {
+        public CalendarService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/calendars";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.